lib/travis/api/v3/services.rb

```diff
@@ -17,6 +17,7 @@ module Services
     Crons         = Module.new { extend Services }
     EnvVar        = Module.new { extend Services }
     EnvVars       = Module.new { extend Services }
+    EnterpriseLicense = Module.new { extend Services }
```

Comment:
May be worth reformatting this area so they all match again in spacing since that was the original setup.